src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import { Todo } from '../../types/Todo';
+import { GlobalLoader } from '../../types/GlobalLoader';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  globalLoader: GlobalLoader,
+};
+
+export const TodoList: React.FC<Props> = (props) => {
+  const { todos, tempTodo, globalLoader } = props;
```

Comment:
Why don't you do destructuring in function parameters?

Reply:
81 symbol

Reply:
@Wesses you can destruct it like that:
```js
export const TodoList: React.FC<Props> = ({ 
  todos, 
  tempTodo, 
  globalLoader 
 }) => { 
  ...
 };
```